mobile_app_connector/models/app_hub.py

```diff
@@ -219,7 +240,6 @@ def _construct_hub_message(
             + "/mobile-app-api"
         )
         hub_endpoint = "/hub/{}?start={}&limit={}"
-        self._assign_order(messages)
```

Comment:
Yes, it is still used. I just changed when it was called (checking it won't change the behaviour), otherwise the tiles would not be sorted when they are stored. This lead to only prayers and donations tiles at the beginning for a few pages.